Dockerfile

```diff
@@ -0,0 +1,30 @@
+# A Docker container for https://github.com/skibish/ddns
+#
+# To build: docker build . -t ddns
```

Comment:
Liked that you have usage block here!

It is better to have syntax `flags arguments`, so it will look like this: `docker build -t ddns .`

```suggestion
# To build: docker build -t ddns .
```